docs/sw-update-conf.json

```diff
@@ -14,110 +14,81 @@
         "prerelease": true
       }
-    }
-  },
-  "mrbeam": {
-    "name": " MrBeam Plugin",
-    "repo": "MrBeamPlugin",
-    "restart": "octoprint",
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeam": {
+      "name": " MrBeam Plugin",
+      "repo": "MrBeamPlugin",
+      "restart": "octoprint",
+      "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
+      "dependencies": {
+        "mrbeam-ledstrips": {
+          "name": "MrBeam LED Strips",
+          "repo": "MrBeamLedStrips",
+          "pip": "https://github.com/mrbeam/MrBeamLedStrips/archive/{target_version}.zip",
+          "global_pip_command": true
+        },
+        "iobeam": {
+          "name": "iobeam",
+          "type": "bitbucket_commit",
+          "repo": "iobeam",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/iobeam.git@{target_version}",
+          "global_pip_command": true,
+          "api_user": "MrBeamDev",
+          "api_password": "v2T5pFkmdgDqbFBJAqrt"
+        },
+        "mrb_hw_info": {
+          "name": "mrb_hw_info",
+          "type": "bitbucket_commit",
+          "repo": "mrb_hw_info",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/mrb_hw_info.git@{target_version}",
+          "global_pip_command": true,
+          "package_name": "mrb-hw-info",
+          "api_user": "MrBeamDev",
+          "api_password": "v2T5pFkmdgDqbFBJAqrt"
         }
       }
-    }
-  },
-  "mrbeamdoc": {
-    "name": "Mr Beam Documentation",
-    "repo": "MrBeamDoc",
-    "pip": "https://github.com/mrbeam/MrBeamDoc/archive/{target_version}.zip",
-    "restart": "octoprint"
-  },
-  "netconnectd": {
-    "name": "OctoPrint-Netconnectd Plugin",
-    "repo": "OctoPrint-Netconnectd",
-    "pip": "https://github.com/mrbeam/OctoPrint-Netconnectd/archive/{target_version}.zip",
-    "restart": "octoprint",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeamdoc": {
+      "name": "Mr Beam Documentation",
+      "repo": "MrBeamDoc",
+      "pip": "https://github.com/mrbeam/MrBeamDoc/archive/{target_version}.zip",
+      "restart": "octoprint"
+    },
+    "netconnectd": {
+      "name": "OctoPrint-Netconnectd Plugin",
+      "repo": "OctoPrint-Netconnectd",
+      "pip": "https://github.com/mrbeam/OctoPrint-Netconnectd/archive/{target_version}.zip",
+      "restart": "octoprint",
+      "dependencies": {
+        "netconnectd-daemon": {
+          "name": "Netconnectd Daemon",
+          "repo": "netconnectd_mrbeam",
+          "pip": "https://github.com/mrbeam/netconnectd_mrbeam/archive/{target_version}.zip",
+          "global_pip_command": true,
+          "package_name": "netconnectd",
+          "branch": "mrbeam2-stable"
```

Comment:
Why is the branch mentioned here but not in the other dependencies?

Reply:
because this branch will always be mrbeam2-stable for all tiers of the legacy image, but I see now that I didn't handle the buster image case, I have to add this